src/markdown-pages/chicken-with-mushrooms-and-caramelized-onions.md

```diff
@@ -0,0 +1,56 @@
+---
+slug: "/recipes/chicken-with-mushrooms-and-caramelized-onions"
+date: "2021-12-12"
+title: "Chicken with mushrooms and caramelized onions"
+mealType: "dinner"
+imageUrl: "https://recipes-md.s3.amazonaws.com/images/no-image-placeholder.svg"
+---
+
+## About the recipe
+
+Serves 4
+
+Total time: 45 mins
+
+Source: [NYT Cooking](https://cooking.nytimes.com/recipes/1022068-skillet-chicken-with-mushrooms-and-caramelized-onions)
+
+## Goes well with
+
+Caesar salad
+
+## Ingredients
+
+- olive oil
+- 2 tablespoons plus 2 teaspoons sherry vinegar
+- 2 teaspoons honey
+- 1 teaspoon Dijon mustard
+- ¼ teaspoon red-pepper flakes
+- Kosher salt and black pepper
+- 1 ½ pounds boneless, skinless chicken thighs, cut into 3-inch pieces
+- 2 medium yellow onions, thinly sliced (about 4 cups)
+- ¾ pound cremini mushrooms, stems removed and thinly sliced (about 4 cups)
+- ½ cup fresh flat-leaf parsley or dill leaves and fine stems, roughly chopped
+- ¼ cup grated Parmesan or pecorino (optional)
+- Bread or cooked pasta, for serving
+
+## Steps
+
+Set a pot of water to boil.
+
+In a large mixing bowl, combine 2 tablespoons oil, 2 tablespoons vinegar, the honey, mustard, red-pepper flakes and 1 teaspoon salt; whisk until smooth. Pat the chicken dry and season with salt and pepper, then add to the mixture, coating it well.
+
+Heat a 12-inch cast-iron over medium-high until very hot, ~2 minutes. Add the onions in an even layer. Season with salt, then cook, mostly undisturbed, for 4 minutes more, stirring every minute or so.
+
+Add the mushrooms, season with salt, and stir to combine. Allow to cook mostly undisturbed until the mushrooms shrink and start to brown, about 4 minutes, stirring every minute or so.
+
+Stir in  3 tablespoons olive oil and allow the onions to cook until they start to color, stirring and lowering the heat as necessary to avoid burning, about 2 minutes.
```

Comment:
```suggestion
Stir in  3 tablespoons of olive oil and allow the onions to cook until they start to color, stirring and lowering the heat as necessary to avoid burning, about 2 minutes.
```